readme.md

```diff
@@ -75,9 +78,6 @@ Position to truncate the string.
 
 ##### space
 
-Type: `boolean`<br>
-Default: `false`
-
```

Comment:
?

Reply:
After checking vs code's git lens I think there was a merge conflict and I used those lines by mistake for preferTruncationOnSpace. I am sorry for messing up PRs so much. I guess I need more experience.